packages/app/src/specs/spec-utils.ts

```diff
@@ -1,14 +1,20 @@
 import fuzzySort from 'fuzzysort'
 import type { FoundSpec } from '@packages/types'
 import { ComputedRef, Ref, ref, watch } from 'vue'
-import _ from 'lodash'
+import _, { cloneDeep } from 'lodash'
 import { FuzzyFoundSpec, getPlatform } from './tree/useCollapsibleTree'
 
 export function fuzzySortSpecs <T extends FuzzyFoundSpec> (specs: T[], searchValue: string) {
-  const normalizedSearchValue = getPlatform() === 'win32' ? searchValue.replaceAll('/', '\\') : searchValue
+  const normalizedSearchValue = normalizeSpecName(searchValue)
+
+  const clonedSpecs = cloneDeep(specs).map((spec) => {
+    return {
+      ...spec,
+      baseName: normalizeSpecName(spec.baseName) }
```

Comment:
We will probably want to apply the same normalization logic to the `relative` property so folders are included.